acme/setup.py

```diff
@@ -29,6 +27,10 @@
         'argparse',
         'mock<1.1.0',
     ])
+elif sys.version_info < (2, 7, 9):
```

Comment:
that matches Python 2.6, so the previously existing code for py2.6 is never run!